content/millennium/r4/financial/support/coverage.md

```diff
@@ -315,13 +315,13 @@ The common [errors] and [OperationOutcomes] may be returned.
 
 ## Delete
 
-Delete an existing Encounter-level Coverage.
+Delete an existing Patient-Level or Encounter-level Coverage.
```

Comment:
can this be "Encounter-evel or Patient level" similar to match with implementation notes. Also level should be lower case for Patient-level as well.

Reply:
Updated - https://github.com/cerner/fhir.cerner.com/pull/733/commits/9b94eec630cc4dc97f496dc3003641e934ff080c